### PR TITLE
New package: motif-2.3.8

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/ddd/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template build file for 'ddd'.
 pkgname=ddd
 version=3.3.12
-revision=5
+revision=6
 build_style=gnu-configure
 makedepends="ncurses-devel libXt-devel libXext-devel libXmu-devel lesstif-devel"
```
Comment (ericonr) [inline srcpkgs/ddd/template:6]: This one still has `lesstif-devel`.

Comment (abenson) [reply srcpkgs/ddd/template]: Yeah, I missed it on it and `xastir`.

File: srcpkgs/xastir/template [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-revision=1
+revision=2
 archs="i686 x86_64 ppc64le"
 wrksrc="Xastir-Release-${version}"
 build_style=gnu-configure
-makedepends="libXrender-devel libXt-devel libXp-devel libcurl-devel pcre-devel lesstif-devel libax25-devel"
+makedepends="libXrender-devel libXt-devel libXp-devel libcurl-devel pcre-devel motif-devel libax25-devel"
 short_desc="X Amateur Station Tracking and Information Reporting"
 maintainer="Michael Aldridge <maldridge@voidlinux.org>"
 license="GPL-2"
```
Comment (mobinmob) [inline srcpkgs/xastir/template:11]: license="GPL-2.0-or-later"
See:  https://github.com/Xastir/Xastir/blob/b17a63bbdfac4422612a99e8acc89c9b4f619340/src/dlm.h#L8-L11

**Comment by mobinmob** [comment]
ddd should have licence="GPL-3.0-or-later" .

File: srcpkgs/motif/template [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	hostmakedepends+=" $makedepends"
+fi
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i lib/Xm/Makefile.am -e 's|$(top_builddir)/config/util/makestr|/usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static &|'
+		vsed -i tools/wml/Makefile.am -e 's|\./wmluiltok|/usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static &|'
+		vsed -i tools/wml/Makefile.am -e 's|\./wml |/usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static &|'
+		vsed -i tools/wml/Makefile.am -e '/motif.wmd:/a	\\tsed -i "s|      exec|& /usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static |" wmldbcreate'
+	fi
```
Comment (ahesford) [inline srcpkgs/motif/template:29]: I suggest moving this block to `post_patch`, so the changes can be verified after `xbps-src patch` wihthout requiring configuration.

File: srcpkgs/motif/template [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+	touch NEWS AUTHORS
+	autoreconf -fi
+	vsed -i tools/wml/wmluiltok.l -e '1i %option main'
```
Comment (ahesford) [inline srcpkgs/motif/template:32]: I believe this can be moved to `post_patch` as well.

**Review by ahesford — COMMENTED** [review]
I built everything for `x86_64*`, `aarch64*`, `armv{6,7}l*` and `i686`, no apparent problems (except a glaring lack of high-dpi support in motif, but what can you do?). `ddd` and `xastir` both launch.

Is CDE next???

**Comment by joshuakraemer** [comment]
I think hostmakedepends is missing "flex".

**Pull request merged**